experimental/Schemes/Auxiliary.jl

```diff
@@ -85,6 +85,7 @@ function pullback(f::AbsCoveredSchemeMorphism, II::AbsIdealSheaf)
   X = domain(f)
   Y = codomain(f)
   scheme(II) === Y || error("ideal sheaf is not defined on the codomain of the function")
+  return PullbackIdealSheaf(f, II)
   phi = covering_morphism(f)
   ID = IdDict{AbsAffineScheme, Ideal}()
   for U in patches(domain(phi))
```

Comment:
Unless I am mistaken, the lines below the newly added line (return `PullbackIdealSheaf(f, II)`) will not be executed. Can those line be removed?

Reply:
Minor nitpick: Support for `pullback_ideal_sheaf` existing?